spec/digital_object_spec.rb

```diff
@@ -617,16 +617,14 @@ def add_object_files(extension='tif')
       expect(ofiles.map { |f| f.relative_path }).to eq(files[m .. -1].sort)
     end
 
-    it "should generate the expected xml text when overriding is explicitly not allowed" do
-      @dobj.project_style[:content_tag_override]=false       # this prevents override of content structure
-      expect(@dobj.content_md_creation_style).to eq(:simple_image)
-      expect(noko_doc(@dobj.content_md_xml)).to be_equivalent_to @exp_xml
-    end
-
-    it "should generate the expected xml text when overriding is not specified" do
-      @dobj.project_style[:content_tag_override]=nil       # this prevents override of content structure
-      expect(@dobj.content_md_creation_style).to eq(:simple_image)
-      expect(noko_doc(@dobj.content_md_xml)).to be_equivalent_to @exp_xml
+    it "should generate the expected xml text when overriding is explicitly not allowed or not specified" do
+     [false, nil].each do |content_tag_override|
```

Comment:
This style is an anti-pattern in test composition.  The more literal tests are more readable and preferable, IMO.

Reply:
Split tests again (but with extra assertions)